### DISCUSSION TITLE
Map Height

### DISCUSSION THREAD

**Discussion by Meso22** [niels747/2D-Weather-Sandbox#50] [Ideas]
It would be cool to have a map height setting where we could change the maps height up to 18000m (60000ft) so we could simulate very high rising tropical cumulonimbus.

**Comment by kidnamednuclearwarin72hours** [comment]
yeah ive noticed that the storms in my simulation often hit the top of the simulation because of the settings i always have

**Comment by 38920** [comment]
what units do the resolution represent

**Reply by kidnamednuclearwarin72hours** [reply]
1 square = 20 * 20 metres from what i understand

**Reply by Meso22** [reply]
I thought 1 cell was 40 meters

**Reply by niels747** [reply]
The height is always 12000 meters, so with vertical res of 300 that is 40m per cell, both horizontally and vertically.

**Reply by niels747** [reply]
I could make the simulation height configurable when creating a new world. Maybe from 8000m to 16000m
Why would you need up to 18000m?
All this would really change is the lapse rate.

**Reply by Meso22** [reply]
I would love to have the sim height configurable to 16000 meters because it helps to form overshooting tops in very powerful storms making them extra realistic!

**Comment by 38920** [comment]
thank you guys for telling me,
